### PR TITLE
[BACKLOG-37742] Page Zoom (AA) - Data Source Wizard Dialog-Database tables subsequent dialogs

### PR DESCRIPTION
**Pull request by varuntangirala**
@pentaho/wcag Please review

This PR is part of 
https://github.com/pentaho/data-access/pull/1210
https://github.com/pentaho/pentaho-analyzer/pull/2447

### CODE REVIEW

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css [hunk 1]
```diff
@@ -1374,6 +1379,49 @@ div#FilterPanel div.pentaho-toggle-button-up button {
...
+  }
+
+  :where(.flex-row).flex-column-sm.flex-center-h {
+    align-items: center;
+    justify-content: unset;
+  }
+
+  :where(.flex-row) .picklistPanel {
+    transform: rotate(90deg);
+  }
```
Comment (dcleao) [inline widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css:1423]: This rule should be together with other rules of the same dialog. Unfortunately, that means repeating the media query statement in other places.

**Review by dcleao — CHANGES_REQUESTED** [review]

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css [hunk 2]
```diff
@@ -5904,14 +5960,18 @@ td.tab-spacer-wrapper {
...
 
 :where(.responsive) :is(.custom-list, .custom-list-disabled) .custom-list-item .gwt-Label {
+  width: 100% !important;
+  overflow: hidden;
   white-space: break-spaces;
+  text-overflow: ellipsis;
+}
+
+:where(.responsive) :is(.custom-list.list-mode, .custom-list-disabled.list-mode) {
+  min-height: 7em;
```
[the remark this answers is not in the document]
Comment (varuntangirala) [reply widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css:5970]: All the list boxes in the database tables and SQL file type needs an some min-height. So previously we used 7em for #connectionList and 6em for #query id which have been removed below in the current PR. Also there are 5 more list boxes(ids: #availableTables, #selectedTables, #leftTables, #rightTables, #joins in main_wizard_panel.xul) which needs an min-height.

[the remark this answers is not in the document]
Comment (dcleao) [reply widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css:5970]: So, approximately, how many list items fit in `7em`?

I believe it is something like:
7em * font-size of 14px / ( measured-line-height of 19px - row-gap of 2px ) ~= 4.5 items

Maybe 3.5 items would be more appropriate as a general default.
3.5 * (19 + 2) / 14 = 5.25em;

Either way, please add a comment above the property mentioning how many line items are expected.

File: widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css [hunk 3]
```diff
@@ -5887,6 +5935,10 @@ td.tab-spacer-wrapper {
   align-items: center;
 }
 
+:where(.responsive) :is(.custom-list, .custom-list-disabled) .custom-list-drop-grid .flex-row {
+  width: 90% !important;
+}
```
[the remark this answers is not in the document]
Comment (dcleao) [reply widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css:5940]: In the "Open File" dialog, the location listbox, in dropdown mode, when with a very long text, wraps into multiple lines. Was that changed in this PR somehow?
If not, what is causing this listbox to behave differently and overflow?

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 9s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **428**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-939/2/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl widgets
```
---
:ok_hand: All tests passed!
 
Tests run: **428**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/gwt-widgets/job/PR-939/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by dcleao — APPROVED** [review]

**Pull request merged**